src/styles/theme.tsx

```diff
@@ -3,11 +3,12 @@ const { width, height } = Dimensions.get("window");
 
 export const COLORS = {
   // base colors
-  primary: "#4840BB",
```

Comment:
@Allen-Muhani @AlphonceMutebi @Chegemartinx 
seems the "primary" base color, and bin line 22 "textBlack" both got replaced. Is that correct???